--- venv/lib/python3.11/site-packages/google_trans_new/google_trans_new.py	2024-02-18 14:51:35
+++ patches/fixed_google_translate.py	2024-02-18 14:54:55
@@ -148,7 +148,7 @@
                 decoded_line = line.decode('utf-8')
                 if "MkEWBc" in decoded_line:
                     try:
-                        response = (decoded_line + '')
+                        response = decoded_line
                         response = json.loads(response)
                         response = list(response)
                         response = json.loads(response[0][2])
